src/commands/getunconfirmed.ts

```diff
@@ -0,0 +1,36 @@
+import { Command, Signup } from '../types'
+import { EmbedBuilder } from 'discord.js'
+import { PermissionFlagsBits } from 'discord.js'
+
+module.exports = new Command({
+	name: 'getunconfirmed',
+	description: 'Gets X unconfirmed users',
+	props: [
+		{ name: 'count', required: false },
+	],
+	allowedChannels: ['bot-commands'],
+	allowedPermissions: PermissionFlagsBits.ManageEvents,
+	async execute({ ia, mongoSignups }) {
+		const unconfirmCount = Math.max(ia.options.getNumber('count') || 5, 10)
+
+		const embed = new EmbedBuilder().setTitle('Unconfirmed users').setTimestamp()
+		if (ia.guild.iconURL()) embed.setThumbnail(ia.guild.iconURL() as string)
+
+		mongoSignups.find({ confirmedBy: "" }, { limit: unconfirmCount }).toArray().then(async (signups: Signup[]) => {
```

Comment:
Found this works too and looks better `const signups = await mongoSignups.find({ confirmedBy: '' }, { limit: unconfirmCount }).toArray()` as it removes a level of nesting